prow/tide/tide_test.go

```diff
@@ -34,16 +34,22 @@ import (
 	apiequality "k8s.io/apimachinery/pkg/api/equality"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/diff"
-	clienttesting "k8s.io/client-go/testing"
+	"k8s.io/client-go/kubernetes/scheme"
+	fakectrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client/fake"
 
 	prowapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
-	"k8s.io/test-infra/prow/client/clientset/versioned/fake"
 	"k8s.io/test-infra/prow/config"
 	"k8s.io/test-infra/prow/git/localgit"
 	"k8s.io/test-infra/prow/github"
 	"k8s.io/test-infra/prow/tide/history"
 )
 
+func init() {
```

Comment:
By putting the production variant of this into `main,go` and not an `init()` in `tide.go` aren't we _very technically_ making it incumbent on importers of Tide methods that use the client to do their own registration? Why don't we do the registration in an `init()` somewhere common?

Reply:
Good point, I could just add an init in the ProwJob api package, then we don't have to explicitly do that everywhere else?